### PR TITLE
Simple development process

### PR DESCRIPTION
**Pull request by OlegAleksashov**

### CODE REVIEW

File: src/api/telegram.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { TOKEN, CHAT_ID } from "../values";
```
Comment (ArturWk4) [inline src/api/telegram.ts:2]: where is _values_ file?

File: src/api/telegram.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+const bsaeUrl = `https://api.telegram.org/bot${TOKEN}/`;
+
+export const sendmessage = async (message: string): Promise<void> => {
```
Comment (ArturWk4) [inline src/api/telegram.ts]: rename it in CamelCase style, for example `sendMessage`

File: src/components/SendForm.jsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+import ModalClose from "@mui/joy/ModalClose";
+import { notifications } from "@mantine/notifications";
+import { validateData } from "../assest/formValidator";
+import { sendmessage } from "../api/telegram";
+
+const SendForm = () => {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [phone, setPhone] = useState("");
+  const [open, setOpen] = useState(false);
```
Comment (ArturWk4) [inline src/components/SendForm.jsx]: it will better to rename to `isOpen`

File: src/api/telegram.ts (hunk 1 above)
Comment (OlegAleksashov) [reply src/api/telegram.ts:2]: I added it to gitignore

Comment (OlegAleksashov) [reply src/api/telegram.ts:2]: I couldn't manage with an error that appeared when I used .env and I created value.js for now

**Pull request merged**